wasm/src/main/java/com/dylibso/chicory/wasm/types/OpCode.java

```diff
@@ -211,7 +211,24 @@ public enum OpCode {
     TABLE_COPY(0xFC0E, List.of(VARUINT, VARUINT)),
     TABLE_GROW(0xFC0F, List.of(VARUINT)),
     TABLE_SIZE(0xFC10, List.of(VARUINT)),
-    TABLE_FILL(0xFC11, List.of(VARUINT));
+    TABLE_FILL(0xFC11, List.of(VARUINT)),
+    V128_LOAD(0xFD00, List.of(VARUINT, VARUINT)),
+    I8x16_SWIZZLE(0xFD0E),
+    V128_CONST(0xFD00 + 13, List.of(VEC_VARUINT)),
```

Comment:
off by one I think:
```bash
$ perl -E 'say 0xFD00 + 13; say 0xFD0C'
64781
64780
```

(some Javy code failed to parse on 0xFD0C which led me to [the table](https://pengowray.github.io/wasm-ops/#simd))